apps/ledger-live-desktop/src/renderer/modals/UpdateFirmwareModal/index.tsx

```diff
@@ -55,6 +58,8 @@ export type StepProps = {
 export type StepId = "idCheck" | "updateMCU" | "updating" | "finish" | "resetDevice" | "restore";
 
 export type Props = {
+  isOpen: boolean;
+  onRequestClose: () => void;
```

Comment:
Also this update on the `Props` definition is causing [this TS error](https://github.com/LedgerHQ/ledger-live/actions/runs/6183570971/job/16785550172#step:6:31)

If we keep this change, i think you need to also update `apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/EarlySecurityChecks/index.tsx` ;)